### PR TITLE
libddwaf v1.8.2

### PR DESCRIPTION
**Pull request by Julio-Guerra**
- [x] Update the libddwaf.a files to v1.8.2
- [x] Introduce `NewHandleFromRuleSet()` 1:1 with `NewHandle()` but now receiving a Go value instead, supposedly the parsed result of the WAF ruleset.
- [x] Remote `UpdateRulesData()` and corresponding tests.

### CODE REVIEW

File: waf_test.go [hunk 1]
```diff
@@ -253,201 +250,9 @@ func TestActions(t *testing.T) {
...
-	test(map[string]interface{}{
-		"http.client_ip": "10.0.0.1",
-		"usr.id":         "moutix",
-	}, []string{"block_user", "block_ip"})
-}
-
 func TestAddresses(t *testing.T) {
 	defer requireZeroNBLiveCObjects(t)
-	expectedAddresses := []string{"my.first.input", "my.second.input", "my.third.input", "my.indexed.input"}
+	expectedAddresses := []string{"my.first.input", "my.second.input", "my.indexed.input", "my.third.input"}
```
Comment (Hellzy) [inline waf_test.go:255]: Not sure I understand this one. Shouldn't `my.third.input` be third?

**Review by Hellzy — APPROVED** [review]
LGTM, thanks for taking care of the stripped .a for darwin 🙏
One small question

File: waf_test.go (hunk 1 above)
Comment (Julio-Guerra) [reply waf_test.go:255]: It's the new output of libddwaf v1.8
cc @Anilm3 if you want to have a look, but I am okay with it.

Comment (Julio-Guerra) [reply waf_test.go:255]: Makes sens, we'll adjust the test next time it breaks, thanks.

**Pull request merged**